plugins/download.py

```diff
@@ -92,6 +90,11 @@ def run(self):
         else:
             pkgs = self._get_pkg_objs_rpms(self.opts.packages)
 
+        if self.opts.destdir:
+            self.conf.destdir = self.opts.destdir
```

Comment:
AttributeError: 'DownloadCommand' object has no attribute 'conf'
Change line to: self.base.conf.destdir = self.opts.destdir